### PR TITLE
Fix VCD datagrid dropdown actions column width

### PR DESCRIPTION
**Pull request by dpresiana**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Examples have been added / updated (for bug fixes / features)
-   [X] Changelog has been updated

## PR Type

What kind of change does this PR introduce?

-   [X] Bugfix
-   [ ] Feature
-   [ ] Code style update (formatting, local variables)
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Build related changes
-   [ ] CI related changes
-   [ ] Documentation content changes
-   [ ] Example website changes
-   [ ] Version bump
-   [ ] Other... Please describe:

## What does this change do?

Minor fix for datagrid dropdown actions column width in some corner cases when datagrid is empty or the columns are rendered before the grid is populated with data

## What manual testing did you do?

Test in the VCD project (no way to test it in the components examples since the issue is a corner case)

Empty state test case steps:
1. Go to Provider Gateway details in the Provider portal with no BGP Community Lists added (should see the empty state)
2. Verify that the column width is normal (38px)

| Before | After |
|--------|--------|
|  Screenshot 2023-06-06 at 19 06 11 | Screenshot 2023-06-06 at 19 05 54 |
 
Columns rendered before data case steps:
1. Go to Content hub in the Tenant portal 
2. Go to Content -> Application Images (this should be opened by default)
3. Switch to see grid view
4. Verify that the column width is normal (38px)

| Before | After |
|--------|--------|
| https://github.com/vmware/vmware-cloud-director-ui-components/assets/77797592/796c3686-77c0-4ca2-a246-f5957c526459 | https://github.com/vmware/vmware-cloud-director-ui-components/assets/77797592/08c42861-989e-4090-85d6-b354d555806f |

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [X] No

## Other information
Bug Number: https://jira.eng.vmware.com/browse/VDU-8198

### CODE REVIEW

File: projects/components/CHANGELOG.MD [hunk 1]
```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- minor fix for datagrid dropdown actions column width in some corner cases when datagrid is empty or the columns are rendered before the grid is populated with data
```
[the remark this answers is not in the document]
Comment (dpresiana) [reply projects/components/CHANGELOG.MD]: Done
